sample-notificator-domain/src/main/groovy/life/qbic/business/notification/refactor/SendEmail.java

```diff
@@ -0,0 +1,51 @@
+package life.qbic.business.notification.refactor;
+
+import java.util.List;
+import java.util.function.Consumer;
+import life.qbic.business.notification.create.NotificationContent;
+
+/**
+ * <p>Implements the business logic of sending emails for project updates.</p>
+ */
+public class SendEmail implements Consumer<List<NotificationContent>>, SendEmailInput{
+    private final EmailSender<NotificationEmail> sendOrRemember;
```

Comment:
What are reasons for emails not being sent? Would an unstable internet connection or so be reasons e.g. is it software related or are there other reasons?